thirdparty/chart-consent-oracle/values.yaml

```diff
@@ -79,7 +79,7 @@ containerSecurityContext:
   runAsUser: 1001
 
 # Add exta environment variables here
-env: {}
+env: []
```

Comment:
Are you sure this is correct @oderayi ?

Reply:
Yes, @kleyow.

That value is an array of objects and not an object. That change fixes a warning printed after `helm install` in previous versions.